extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -2310,6 +2309,40 @@ export class AmpA4A extends AMP.BaseElement {
     }
     return null;
   }
+
+  /**
+   * Returns any enabled SSR experiments via the amp-usqp meta tag. These
+   * correspond to the proto field ids in cs/AmpTransformerParams.
+   *
+   * These experiments do not have a fully unique experiment id for each value,
+   * so we concatenate the key and value to generate a psuedo id. We assume
```

Comment:
Should we create real exp id for these ssr features? I am not sure this pseudo id is permissible.